olp-cpp-sdk-core/src/http/Network.cpp

```diff
@@ -52,6 +52,14 @@ std::shared_ptr<Network> CreateDefaultNetworkImpl(size_t max_requests_count) {
 }
 }  // namespace
 
+void Network::SetDefaultHeaders(Headers /*headers*/) {}
+
+bool Network::SetCurrentBucket(uint8_t /*bucket_id*/) { return false; }
+
+Network::Statistics Network::GetStatistics(uint8_t /*bucket_id*/) {
```

Comment:
```suggestion
Network::Statistics Network::GetStatistics(uint8_t) {
```

Reply:
a variable name is placed for consistency